xstream/src/test/com/thoughtworks/xstream/io/naming/NoNameCoderTest.java

```diff
@@ -0,0 +1,32 @@
+package com.thoughtworks.xstream.io.naming;
+
+import com.thoughtworks.xstream.io.naming.NoNameCoder;
+import junit.framework.TestCase;
+
+public class NoNameCoderTest extends TestCase {
+
+  public void testConstructor() {
+    final NoNameCoder noNameCoder = new NoNameCoder();
```

Comment:
Unneeded test for synthetic constructor (I think)

Reply:
Yeah i've removed it. Coverage still the same.